docroot/profiles/custom/sitenow/sitenow.install

```diff
@@ -964,3 +964,38 @@ function sitenow_update_8029() {
 
   }
 }
+
+/**
+ * Roll back contact block revision on collegiate sites.
+ */
+function sitenow_update_8030() {
+  /** @var \Drupal\Core\Plugin\DefaultPluginManager $filters */
+  $filters = \Drupal::service('plugin.manager.config_filter')->getDefinitions();
+  $collegiate_split = 'config_split:collegiate';
+  // This site has the 'collegiate' split enabled.
+  if (isset($filters[$collegiate_split]) && $filters[$collegiate_split]['status']) {
```

Comment:
FYI `\Drupal::configFactory()->getEditable('config_split.config_split.collegiate')->get('status');` also seems to work here. I wasn't sure it would since status is ignored.

https://github.com/uiowa/uiowa/blob/deb4c8c179a37fd553ad54cb7da230339ff54769/config/default/config_ignore.settings.yml#L6

Reply:
Thanks for this, @richardbporter. This might help me resolve the issues I had run into in #1932.